src/Bowerphp/Command/InitCommand.php

```diff
@@ -13,12 +13,14 @@
 
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpKernel\Kernel;
```

Comment:
Too bad HttpKernel is **not** in our dependencies.
You should rely on feature detect, not on declared versions.